properties.py

```diff
@@ -469,11 +473,11 @@ def reset_ll_object_index(self, context):
         min=-32, max=32, default=-1)
     max_trace_depth = IntProperty(
         name="Max Trace Depth",
-        description="Maximum number of ray bounces (0 disables ray tracing)",
+        description="Maximum number of times a ray can bounce before termination.\nLower settings will render faster, but may also dull bounce lighting.",
         min=0, max=32, default=10)
     max_specular_depth = IntProperty(
         name="Max Specular Depth",
-        description="Maximum number of specular ray bounces",
+        description="Maximum number of specular (reflective) ray bounces",
```

Comment:
No, it's not necessary.  I had seen it referred to that way in some wiki docs so I tossed it in.